hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/metrics/FlinkCompactionMetrics.java

```diff
@@ -66,6 +66,12 @@ public class FlinkCompactionMetrics extends FlinkWriteMetrics {
    */
   private long compactionCost;
 
+  /**
+   * flag indicates whether rollback has occurred in final compaction state
+   */
```

Comment:
`flag indicates whether rollback has occurred in final compaction state` ->
`Flag saying whether the compaction is completed or been rolled back.`